python/docs/source/conf.py

```diff
@@ -22,6 +22,10 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath('.'))
 
+# generate user_guide/pandas_on_spark/supported_pandas_api.rst
+from user_guide.pandas_on_spark.supported_status_rst_generator import SuppportedStatusRSTGenerator
+SuppportedStatusRSTGenerator().execute()
```

Comment:
BTW, if we're going to generate this ways, maybe we should better remove https://github.com/apache/spark/blob/master/python/docs/source/user_guide/pandas_on_spark/supported_pandas_api.rst